components/identity.oauth.uma.dbscripts/uma_tables.sql

```diff
@@ -1,17 +1,18 @@
 CREATE TABLE IDN_RESOURCE (
-  ID                INTEGER   AUTO_INCREMENT PRIMARY KEY NOT NULL,
-  RESOURCE_ID       VARCHAR(255),
-  RESOURCE_NAME     VARCHAR(30),
-  TIME_CREATED      TIMESTAMP DEFAULT '0',
-  RESOURCE_OWNER_ID VARCHAR(255),
-  TENANT_ID         INTEGER
+  ID                  INTEGER   AUTO_INCREMENT PRIMARY KEY NOT NULL,
+  RESOURCE_ID         VARCHAR(255),
+  RESOURCE_NAME       VARCHAR(255),
+  TIME_CREATED        TIMESTAMP DEFAULT '0',
+  RESOURCE_OWNER_NAME VARCHAR(255),
```

Comment:
Shall we use Resource_owner or resource_owner_id here?

Reply:
From the PAT validation filter we can obtain the resource_owner_name. Therefore, should we add a db call to obtain the relevant resource_owner_id?